.github/workflows/check-runtime-migration.yml

```diff
@@ -55,28 +55,33 @@ jobs:
             uri: "wss://try-runtime-westend.polkadot.io:443"
             subcommand_extra_args: "--no-weight-warnings --blocktime 6000"
             command_extra_args: ""
+            checks: "all-except-Staking,NominationPools,FastUnstake,DelegatedStaking"
```

Comment:
I may have been wrong, the thing that gets passed to the try-runtime CLI is: `UpgradeCheckSelect`, not `Select`.